dask/dataframe/groupby.py

```diff
@@ -1775,6 +1790,9 @@ def aggregate(self, arg, split_every=None, split_out=1, shuffle=None):
                 f"if pandas < 1.1.0. Pandas version is {pd.__version__}"
             )
 
+        if self.sort is None and split_out > 1:
+            warnings.warn(SORT_SPLIT_OUT_WARNING, FutureWarning)
+
         if shuffle:
```

Comment:
`self.sort` can be `True` if we use the shuffle code path below. Maybe just move this after the `if shuffle` code block?

The shuffle code path below will throw a slightly different error if the number of groupby columns is >1. We could either leave that error as is, or use `SORT_SPLIT_OUT_WARNING` there as well (and update it to mention the shuffle-based code path).

Reply:
Yeah, I was trying to cover both code paths with this warning, but you are correct that the condition is slightly different for the shuffle case. I can move things around to be more specific.